testsuite/feast.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+import pickle
+import shutil
+import tempfile
+import unittest
+
+from @LIBNAME@ import picomplex as pi
+
+
+class FeastTest(unittest.TestCase):
+
+    def setUp(self):
+        # Set up cache
+        self.cache = pi.MatrixElementCache()
+
+    def test_diagonalization_full(self):
+        if not pi.mkl_enabled:
```

Comment:
You can use the [`@unittest.skipIf`](https://docs.python.org/3.6/library/unittest.html#unittest.skipIf) decorator for the `FeastTest` class.  This will also mark the test properly as skipped.